quiz.js

```diff
@@ -0,0 +1,57 @@
+
+document.getElementById("goButton").addEventListener("click", makeTree);
+
+document.addEventListener('keyup', function (event) {
+  if (event.which === 13) {
+    makeTree();
```

Comment:
This function call should be taking an object that is comprised of the char and height.